en/docs/using-wso2-identity-server/setting-up-oauth-token-hashing.md

```diff
@@ -19,35 +16,28 @@ authorization codes.
 
 Follow the instructions below to set up OAuth token hashing:
 
-1.  Edit the
-    `          <IS_HOME>/repository/conf/identity/identity.xml         `
-    file,and do the following configuration changes under the
-    `          <OAuth>         ` section:
-    -   Change the value of the
-        `             <TokenPersistenceProcessor>            ` element
-        as follows to enable token hashing:
-
-        ``` xml
-        <TokenPersistenceProcessor>org.wso2.carbon.identity.oauth.tokenprocessor.HashingPersistenceProcessor</TokenPersistenceProcessor>
+1.  Do the following configurations in the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder.
+    -   Add the following token persistence processor to  enable token hashing:
+        ``` toml
+        [oauth.extensions]
+        token_persistance_processor = "org.wso2.carbon.identity.oauth.tokenprocessor.HashingPersistenceProcessor"
         ```
 
-    -   Change the value of the
-        `             <EnableClientSecretHash>            ` element to
-        `             true            ` as follows:
-
-        ``` xml
-                <EnableClientSecretHash>true</EnableClientSecretHash>
+    -   Add the following property and set it to true to enable hashing. 
+        ``` toml
+        [oauth]
+        hash_tokens_and_secrets= true
         ```
 
     -   Add the following configuration to specify the algorithm to use
         for hashing:
 
-        ``` xml
-                <HashAlgorithm>SHA-256</HashAlgorithm>
+        ``` toml
+        [oauth]
+        hash_token_algorithm= ""
```

Comment:
Isn't SHA-256 the value?

Reply:
fixed with 8fce9b7e60b0c67e606677f9fc5d06b1612c3ae8